### ISSUE TITLE
Installation Error

### ISSUE THREAD

**Issue by joshuachen1122** [tensorflow/tensorflow#28893]
**System information**
- Windows 10
- TensorFlow installed from (source or binary): pip
- TensorFlow version: Alpha 2.0
- Python version: 3.7.3
- Installed using: pip
- GPU model and memory: GTX 1060, 16GB ram

My import tensorflow statement is not working. Here is the entire stack trace:

```
(base) D:\Project>python tf.py
Traceback (most recent call last):
  File "C:\Users\Joshua\Anaconda3\lib\site-packages\tensorflow\python\pywrap_tensorflow.py", line 58, in <module>
    from tensorflow.python.pywrap_tensorflow_internal import *
  File "C:\Users\Joshua\Anaconda3\lib\site-packages\tensorflow\python\pywrap_tensorflow_internal.py", line 28, in <module>
    _pywrap_tensorflow_internal = swig_import_helper()
  File "C:\Users\Joshua\Anaconda3\lib\site-packages\tensorflow\python\pywrap_tensorflow_internal.py", line 24, in swig_import_helper
    _mod = imp.load_module('_pywrap_tensorflow_internal', fp, pathname, description)
  File "C:\Users\Joshua\Anaconda3\lib\imp.py", line 242, in load_module
    return load_dynamic(name, filename, file)
  File "C:\Users\Joshua\Anaconda3\lib\imp.py", line 342, in load_dynamic
    return _load(spec)
ImportError: DLL load failed: The specified module could not be found.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "tf.py", line 13, in <module>
    import tensorflow as tf
  File "C:\Users\Joshua\Anaconda3\lib\site-packages\tensorflow\__init__.py", line 24, in <module>
    from tensorflow.python import pywrap_tensorflow  # pylint: disable=unused-import
  File "C:\Users\Joshua\Anaconda3\lib\site-packages\tensorflow\python\__init__.py", line 49, in <module>
    from tensorflow.python import pywrap_tensorflow
  File "C:\Users\Joshua\Anaconda3\lib\site-packages\tensorflow\python\pywrap_tensorflow.py", line 74, in <module>
    raise ImportError(msg)
ImportError: Traceback (most recent call last):
  File "C:\Users\Joshua\Anaconda3\lib\site-packages\tensorflow\python\pywrap_tensorflow.py", line 58, in <module>
    from tensorflow.python.pywrap_tensorflow_internal import *
  File "C:\Users\Joshua\Anaconda3\lib\site-packages\tensorflow\python\pywrap_tensorflow_internal.py", line 28, in <module>
    _pywrap_tensorflow_internal = swig_import_helper()
  File "C:\Users\Joshua\Anaconda3\lib\site-packages\tensorflow\python\pywrap_tensorflow_internal.py", line 24, in swig_import_helper
    _mod = imp.load_module('_pywrap_tensorflow_internal', fp, pathname, description)
  File "C:\Users\Joshua\Anaconda3\lib\imp.py", line 242, in load_module
    return load_dynamic(name, filename, file)
  File "C:\Users\Joshua\Anaconda3\lib\imp.py", line 342, in load_dynamic
    return _load(spec)
ImportError: DLL load failed: The specified module could not be found.


Failed to load the native TensorFlow runtime.
```

**Comment by abhinavsagar** [comment]
How did you install tensorflow ?

**Comment by joshuachen1122** [comment]
I solved the issue! I installed tensorflow 2.0, which I understand is currently in alpha. I was running into issues with tensorboard and I accidentally uninstalled the wrong version. So i fixed it by uninstalling everything and installing an older, more stable version of tensorflow. Thanks for the response.

**Comment by achandraa** [comment]
Good to know the issue is resolved.
